tc-redirect-tap/pkg.yaml

```diff
@@ -12,10 +12,9 @@ steps:
     prepare:
       - |
         tar -xzf tc-redirect-tap.tar.gz --strip-components=1
+  - network: default
```

Comment:
I wonder if we should have a step with just `go mod download -readonly` and then the build step without network access

Reply:
@smira wdyt?

Reply:
should improve it without considerable drawbacks. let me try